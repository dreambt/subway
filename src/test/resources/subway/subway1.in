L3-2,L1-3