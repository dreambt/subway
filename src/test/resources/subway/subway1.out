2:L3-2,L3-3,X1,L1-3